xdebug_com.c

```diff
@@ -23,7 +23,11 @@
 #include <stdio.h>
 #include <fcntl.h>
 #ifndef PHP_WIN32
-# include <sys/poll.h>
+# if HAVE_SYS_POLL_H
+#  include <sys/poll.h>
+# elif HAVE_POLL_H
+#  include <poll.h>
+# endif
```

Comment:
The musl libc will have sys/poll.h also but with a warning, so it might be better to replace that if elif check into this:

```
# if HAVE_POLL_H
#  include <poll.h>
# elif HAVE_SYS_POLL_H
#  include <sys/poll.h>
# endif
```

Thank you @derickr for this patch